recipes/luarocks/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+
+
+# TODO
+# - Would be good if luarocks would offer the ability to use only a user rockspec, instead of system.
+#   (Using system-only rockspec with --force-config flag installs packages with restrictive privileges)
+./configure --prefix=$PREFIX \
+    --sysconfdir=$PREFIX/share/lua/ \
+    --with-lua-include=$PREFIX/include \
+    --rocks-tree=$PREFIX
+
+make bootstrap
```

Comment:
[stack overflow](http://stackoverflow.com/questions/29334171/should-icompile-luarocks-as-usual-or-as-a-rock) tells me that `make bootstrap` installs luarocks as a rock, and can then be upgraded with `luarocks install luarocks`. 

So it seems that we should stick with `make build; make install` in order to stick with the version supplied by conda.